app/src/controllers/reservation/repositories/ShopRepository.ts

```diff
@@ -87,7 +87,12 @@ const ShopRepository: ReservationServiceSocket & ShopServiceSocket = {
         shopDetail: true,
       },
     })
-    return shop ? { startTime: shop.shopDetail.startTime, endTime: shop.shopDetail.endTime } : null
+    return shop ? {
+      startTime: shop.shopDetail.startTime,
+      endTime: shop.shopDetail.endTime,
+      seats: shop.shopDetail.seats,
```

Comment:
if you're also returning seats, i think you should change the name of the method.

Reply:
i think you need to check if within the same days as well